### PR TITLE
N27: Added Hydrofoil Generation Sailing

### PR DESCRIPTION
**Pull request by kunos**
_Part of #818_

### CODE REVIEW

File: content/news/027/index.md [hunk 1]
```diff
@@ -68,6 +68,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
 ## Game Updates
 
+### [Hydrofoil Generation Sailing]
+
+![hgs_screen](hydrofoil1_1024.png)
+
+[Hydrofoil Generation Sailing] is a realistic sailing/foiling inshore simulator
+in development for PC/Steam that will put you in the driving seat of modern
+competitive sailing.
+The game is the brain child of industry veteran Stefano Casillo
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:78]:
```suggestion

The game is the brain child of industry veteran Stefano Casillo
```

File: content/news/027/index.md [hunk 2]
```diff
@@ -68,6 +68,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+(of Assetto Corsa fame) and features a custom made 3D engine based on DirectX11
+via winapi-rs.
+An engine conversion to WGPU is currently under evaluation in order to
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:81]:
```suggestion

An engine conversion to WGPU is currently under evaluation in order to
```

File: content/news/027/index.md [hunk 3]
```diff
@@ -68,6 +68,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+guarantee an easier port to platform such as Steam Deck or even mobile in the
+future.
+
+Early Access release on Steam is expected in mid 2022.
+You can follow the project on:
+
+- [Facebook](https://www.facebook.com/HydrofoilGenerationSailing)
+- [Discord](https://discord.gg/DtKgt2duAy)
+
+[Hydrofoil Generation Sailing]: https://www.facebook.com/HydrofoilGenerationSailing/
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:91]:
```suggestion
Early Access release on Steam is expected in mid 2022.
You can follow the project on [Facebook](https://www.facebook.com/HydrofoilGenerationSailing)
and [Discord](https://discord.gg/DtKgt2duAy).

[Hydrofoil Generation Sailing]: https://www.facebook.com/HydrofoilGenerationSailing/
```

**Review by 17cupsofcoffee — APPROVED** [review]
:+1: Looks good to me, just going to add a bit of extra spacing for readability,.

File: content/news/027/index.md [hunk 4]
```diff
@@ -68,6 +68,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+The game is the brain child of industry veteran Stefano Casillo
+(of Assetto Corsa fame) and features a custom made 3D engine based on DirectX11
+via winapi-rs.
+
+An engine conversion to WGPU is currently under evaluation in order to
+guarantee an easier port to platform such as Steam Deck or even mobile in the
+future.
+
+Early Access release on Steam is expected in mid 2022.
+You can follow the project on [Facebook](https://www.facebook.com/HydrofoilGenerationSailing)
```
Comment (ozkriff) [inline content/news/027/index.md:88]: maybe move these fb/discord to a braced list in the first sentence?

Comment (17cupsofcoffee) [reply content/news/027/index.md]: Done.

**Pull request merged**